templates/subject.html

```diff
@@ -83,7 +83,7 @@
     {% endif %}
 
         <!--./hot article end-->
-    {% for article in articles[::-1] %}
+    {% for article in articles.items %}
```

Comment:
What is the difference between these two?, what does `articles.items` mean ?.

Thanks.

Reply:
@hema-001 
As we've established above, the result we get when we query the articles is not a list object anymore, since we're now using `paginate()`. The line I removed was for iterating over a list object and I replaced it with the one for iterating over items (articles) on a single page. You can see from the screenshot below that `all()` returns a list object and `pagingate()` returns a `flask_sqlalchemy.Pagination` object.

![image](https://user-images.githubusercontent.com/45811924/79834462-b2447d00-83df-11ea-9235-b5c216f10584.png)

`articles.items` is a list of articles on a single page.